README.md

```diff
@@ -1,9 +1,66 @@
-# triton-docker
+# Triton Docker CLI helper
 
-Docker CLI helper for Triton
+This script installs known-good, tested versions of the Docker (now Moby) and Docker Compose CLI tools for use with Triton.
+
+Additionally, this script will automatically configure those tools for use with Triton when you use them, making it easy to switch between Docker on your laptop and Docker on Triton.
+
+### Compatibility and requirements
+
+This is designed and tested for Linux and MacOS X.
+
+This code also requires [the Triton CLI tools](https://docs.joyent.com/public-cloud/api-access/cloudapi) and [a Triton account](https://docs.joyent.com/public-cloud/getting-started) on either the Triton Public Cloud (Joyent public cloud) or in a private cloud powered by Triton.
+
+Use of this software also requires [a Triton profile configured in the Triton CLI tool](https://docs.joyent.com/public-cloud/api-access/cloudapi#configuration).
 
 ### Installation
 
+In a terminal window, run the following command:
+
 ```bash
-curl -o /usr/local/bin/triton-docker https://raw.githubusercontent.com/misterbisson/triton-docker/working/triton-docker && chmod +x /usr/local/bin/triton-docker && ln -Fs /usr/local/bin/triton-docker /usr/local/bin/triton-compose && ln -Fs /usr/local/bin/triton-docker /usr/local/bin/triton-docker-install
+sudo curl -o /usr/local/bin/triton-docker https://raw.githubusercontent.com/joyent/triton-docker-cli/master/triton-docker && chmod +x /usr/local/bin/triton-docker && ln -Fs /usr/local/bin/triton-docker /usr/local/bin/triton-compose && ln -Fs /usr/local/bin/triton-docker /usr/local/bin/triton-docker-install
 ```
+
+That command will copy the `triton-docker` shell script from this repo, and link it as `triton-compose` and `triton-docker-install`.
+
+To complete the installation, run `sudo triton-docker-install` to install the platform-specific versions of the Docker (now Moby) and Docker Compose CLI tools. These versions will not replace any existing Docker or Docker Compose versions you may have installed.
+
+### Usage
+
+Once installed, use `triton-docker` and `triton-compose` in place of `docker` and `docker-compose` when interacting with the Triton Elastic Docker Host.
```

Comment:
There's no mention of the lack of need to use `eval "$(triton env)"`. On one hand, it doesn't hurt for people to get into the environment they want to be in, regardless of the need. On the other, it is a nice step-cutting addition.